app/views/docs/upgrade.phtml

```diff
@@ -1,19 +1,40 @@
 </div>
 
 <!-- <p>When updating to a new Appwrite version, make sure to run the <a href="/docs/installation">installation script for the new version</a> from the same location, you executed your original setup. This will allow the installation script to update your existing docker-compose.yml file. The correct location for execution is the parent directory of your existing appwrite directory.</p> -->
 
 <!-- <p>The data migration tool will allow you to easily migrate your current Appwrite data to work with the new version. </p> -->
 
-<p>To start the migration, run the following command from your terminal once your backup is ready and that your new Appwrite setup has been completed successfully. Replace "[APPWRITE_CONTAINER_ID]" with your container ID. You can find your container ID by running `docker ps` command.</p>
+<p> Head over to the directory containing your <b>docker-compose.yml</b> . If Appwrite is currently running, your first step would be to shut it down using: </p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose down</code></pre>
+</div>
 
+<p>Next you need to go to the parent directory and install the latest version of Appwrite</p>
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose exec appwrite migrate</code></pre>
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>
+    cd ../
+
+    docker run -it --rm \
+    --volume /var/run/docker.sock:/var/run/docker.sock \
+    --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
+    --entrypoint="install" \
+    appwrite/appwrite:<?php echo APP_VERSION_STABLE; ?></code></pre>
 </div>
+
+<p>Once the setup completes, verify that you have the latest version of Appwrite.</p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker ps | grep appwrite/appwrite</code></pre>
+</div>
+<p>Verify that the <b>STATUS</b> doesn't have any errors and all the <b>appwrite/appwrite</b> containers have version : <?php echo APP_VERSION_STABLE; ?></p>
+
+<p>We can now start the migration. Navigate to the <b>appwrite</b> directory where your <b>docker-compose.yml</b> is present and run the following command</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose exec appwrite migrate</code></pre>
+</div>
```

Comment:
Maybe we should add an ending line? like once the setup has completed your Appwrite data is ready... you can now...